src/screens/LogIn/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { View, Text } from 'react-native'
+
+const index = () => {
+    return (
+        <View>
+            <Text>LogIN</Text>
+        </View>
+    )
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file.